index.html

```diff
@@ -155,6 +157,104 @@ <h1 class="text-center text-info" id="news">News</h1>
             </ul>
           </div>
         </div>
+        <div class="carousel slide" id="news-carousel" data-interval="5000" data-ride="carousel">
```

Comment:
i cannot use the mouse to navigate the slides (on local version), it only works with the arrow keys. Is it normal?